docs/sql-guide/functions/function-cosine-distance.md

```diff
@@ -7,20 +7,25 @@ grand_parent: SQL guide
 # COSINE_DISTANCE() function
 
 `COSINE_DISTANCE()` is a mathematical function that can be performed on `VECTOR` data type columns.
+The cosine distance, sometimes called the cosine similarity, of two vectors is a measure of the
+angle between them. This gives a measure of similarity ignoring the magnitude of the vectors.
 
 ## Syntax
 
 ```sql
-COSINE_DISTANCE (<compare-vector>, <vector-column>)
+COSINE_DISTANCE (<vector1>, <vector2>)
```

Comment:
Call me picky (go ahead, really, because I am!), but could this be represented as:
```
COSINE_DISTANCE ([<vector-literal> | <vector-col>],...)
```
I say this to be absolutely clear rather than relying on the arguments table which repeats the same thing for both vector1 and vector2 :D